site/app/templates/submission/regrade/Discussion.twig

```diff
@@ -1,6 +1,46 @@
+<div><h3>Regrade Request
+{% if btn_type == "request" %}
+    </h3></div>
+    <div><p>"{{ regradeMessage }}"</p></div>
+    {% set message = "Request Regrade" %}
+{% elseif btn_type == "admin_open" %}
+    <span><span class="fa fa-hourglass" style="font-size: 16px; color: #af0000"></span> 
+    <span class="red-message">Regrade Request Awaiting Review</span></span> 
+    </h3></div>
+    {% set message = "Type response here" %}
```

Comment:
Can you reindent / reformat this code? It's basically impossible to see the structure if it's all bunched together like this